featuretools/tests/entityset_tests/test_es.py

```diff
@@ -119,6 +119,18 @@ def test_add_relationship_errors_on_dtype_mismatch(es):
         es.add_relationship(mismatch)
 
 
+def test_add_relationship_empty_child_convert_dtype(es):
```

Comment:
can this test also have an assert to confirm that the colum data type was not `int64` to begin with?